java/code/src/com/suse/manager/webui/Router.java

```diff
@@ -82,13 +83,17 @@ public void init() {
 
         initNotFoundRoutes(jade);
 
+        TaskomaticApi taskomaticApi = new TaskomaticApi();
```

Comment:
Nitpick: Please move this line with other _APIs instance creations, it will improve the readability. It's not bad but for example, in the case of `StatesAPI` on line 95, I need to look where does this `taskomaticApi` instance come from.
 We could even inline `taskomaticApi` instance if it is being used only once in the class and that a parameter of `StatesAPI` 's constructor.

Reply:
The naming is a bit confusing here. Everything above line 90 are general instances independent of WebAPI/UI while everything below are http endpoints where *API means api endpoints only and *Controller are usually api/ui endpoints.
I think we either need to make the distinction clearer or get rid of it.